api_tests/tests/peers_using_ip.ts

```diff
@@ -1,48 +1,43 @@
 import { threeActorTest, twoActorTest } from "../src/actor_test";
 import { createDefaultSwapRequest, sleep } from "../src/utils";
-import { expect, request } from "chai";
 import { Actor } from "../src/actors/actor";
 
 // ******************************************** //
 // Peers using ips                              //
 // ******************************************** //
 
-async function assertNoPeersAvailable(actor: Actor, message: string) {
-    const peersResponse = await request(actor.cndHttpApiUrl()).get("/peers");
+async function assertNoPeersAvailable(actor: Actor) {
+    const peersResponse = await actor.cnd.fetch("/peers");
+    const body = peersResponse.data as {
+        peers: { id: string; endpoints: string[] }[];
+    };
 
-    expect(peersResponse.status).to.equal(200);
-    expect(peersResponse.body.peers, message).to.be.empty;
+    expect(peersResponse.status).toBe(200);
+    expect(body.peers).toHaveLength(0);
 }
 
-async function assertPeersAvailable(alice: Actor, bob: Actor, message: string) {
-    const peersResponse = await request(alice.cndHttpApiUrl()).get("/peers");
+async function assertPeersAvailable(alice: Actor, bob: Actor) {
```

Comment:
:thinking:  What about a single function that allows you to pass an array of actors, so that you can assert that all of the elements of the array exist in the list of peers returned by cnd? Then you could call `assertPeers(actor, [])` replacing `assertNoPeersAvailable`?

Reply:
If you don't agree with the above proposal, then this function's argument names should change to something like:

```
alice -> actor OR self
bob -> peer OR candidate
```

Reply:
```
Error: expect(received).toContainEqual(expected) // deep equality

Expected value: []
Received array: []
```

Well that's sad.

Reply:
https://i.redd.it/ss8t1h4q89d31.png

Reply:
```
Error: expect(received).toContainEqual(expected) // deep equality

Expected value: [{"id": "QmRyWYuMnXM8Jv9zPNuuC8tLxx8bYVrMt7HFGXd5A6eGnQ"}]
Received array: [{"id": "QmRyWYuMnXM8Jv9zPNuuC8tLxx8bYVrMt7HFGXd5A6eGnQ"}]
```
Seriously?

Reply:
aaaah, I just realised that I should pass an item in the function, not a full list to compare!!!